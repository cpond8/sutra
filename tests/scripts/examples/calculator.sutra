;; Simple calculator example
;; Demonstrates basic arithmetic operations

(print "=== Simple Calculator ===")
(print (+ 10 5))
(print (- 20 8))
(print (* 4 7))
(print (/ 36 6))
(print (mod 17 5))

(print "=== Comparisons ===")
(print (gt? 10 5))
(print (lt? 3 8))
(print (eq? 4 4))

(print "=== Logic ===")
(print (not true))
(print (not false))
