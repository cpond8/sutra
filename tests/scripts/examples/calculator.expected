=== Simple Calculator ===
15
12
28
6
2
=== Comparisons ===
true
true
true
=== Logic ===
false
true
